sysvinit/openbsd/cloudconfig.tmpl

```diff
@@ -0,0 +1,39 @@
+## template:jinja
+#!/bin/ksh
+
+# PROVIDE: cloudconfig
+# REQUIRE: cloudinit
+# BEFORE: sshd
+
+
+daemon="cloud-init"
+daemon_execdir="{{prefix}}/bin"
+daemon_flags="modules --mode config"
+daemon_rtable=0
+daemon_timeout=1
```

Comment:
from [rc.subr(8)](https://man.openbsd.org/rc.subr.8)

> #### daemon_timeout
>    Maximum time in seconds to wait for the start, stop and reload actions to return. This is only guaranteed with the default rc_start, rc_stop and rc_reload functions.

so, yeah, i think it needs to go.
provided we keep these service files to begin with.

OpenBSD's rc can only really manage services that remain running. It's not very good at one-shot tasks.
We might be able to fake it by having rc_check always return true, but i don't know how well that will actually work. because it doesn't give us a guarantee that a stage is finished

Reply:
I will test removing this and see how it responds without setting it.  Even though it is set to 1, it returns an (ok) from the start, but because the daemon_bg is set, the process still runs in the background and I'm able to verify this with running a check of the service.  The check returns (ok) while the service is running and as soon as it completes, the check of the service returns (failed).

Reply:
> I will test removing this and see how it responds without setting it. Even though it is set to 1, it returns an (ok) from the start, but because the rc_bg is set, the process still runs in the background and I'm able to verify this with running a check of the service. The check returns (ok) while the service is running and as soon as it completes, the check of the service returns (failed).

Right, that's what I'm not so clear on - whether the `daemon_timeout` applies to the daemon in the background (within the call to `rc_exec`) or to the process of putting the daemon in the background (within the call to `rc_start`). If the former, I think this is a problem, but if the latter, then I would think this is fine (so long as cloud-init can keep running in the background after `rc_start` exits).

Reply:
All tests seemed fine with removing `daemon_timeout` and `daemon_rtable`